ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1451,4 +1452,47 @@ FCU_RTC_RHDHS_DTM_DFSS:
   - master_alarm
   - water_leak_alarm
   - fire_alarm
-  - schedule_run_command
\ No newline at end of file
+  - schedule_run_command
+  
+#PAC Unit #  
+FCU_CHWDC_DFSS_DXSC_DXZTC_EVFSM_REFPM_ZHC_DFRMM_CHWDT:
+  description: "CRAC with chilled water temperature monitoring and compressor monitoring with zone temperature and humidity monitoring."
+  is_canonical: true
+  implements:
+  - FCU
+  - CHWDC
+  - DFSS
+  - DXSC
+  - DXZTC
+  - EVFSM
```

Comment:
explain this. evaporative fan separate from the discharge fan? the evaporator fan in a typical crac unit is the discharge fan. how is this separate?

Reply:
EFVSM has been replaced by existing DFVSC and the associated telemetries have been deleted.